internal/provider/resource_ssh_key_test.go

```diff
@@ -0,0 +1,127 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/oxidecomputer/oxide.go/oxide"
+)
+
+type resourceSSHKeyConfig struct {
+	BlockName   string
+	Name        string
+	Description string
+	PublicKey   string
+}
+
+var resourceSSHKeyConfigTpl = `
+resource "oxide_ssh_key" "{{.BlockName}}" {
+  name        = "{{.Name}}"
+  description = "{{.Description}}"
+  public_key  = "{{.PublicKey}}"
+  timeouts = {
+    read   = "1m"
+    create = "3m"
+    delete = "2m"
+    update = "4m"
+  }
+}
+`
+
+var resourceSSHKeyUpdateConfigTpl = `
+resource "oxide_ssh_key" "{{.BlockName}}" {
+  name        = "{{.Name}}"
+  description = "{{.Description}}"
+  public_key  = "{{.PublicKey}}"
+  timeouts = {
+    read   = "1m"
+    create = "3m"
+    delete = "2m"
+    update = "4m"
+  }
+}
+`
```

Comment:
Looks like this needs to be removed as well

Reply:
Oops! Fixed it now.